eng/SourceBuild.props

```diff
@@ -49,6 +49,7 @@
       <InnerBuildArgs>$(InnerBuildArgs) /p:EnableNgenOptimization=false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:AdditionalRuntimeIdentifierParent=$(BaseOS)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) /p:EnablePackageValidation=false</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(SourceBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) /p:PrimaryRuntimeFlavor=Mono /p:RuntimeFlavor=Mono</InnerBuildArgs>
```

Comment:
I always forget what the difference between these two properties is. Can you please add a comment describing the set values?

Reply:
These property names are not intuitive. If you build a .NET SDK, `PrimaryRuntimeFlavor` and `RuntimeFlavor` have to have the same value. So either both are `Mono` or both are `CoreCLR`.

I don't think I can come up with a meaningful comment that doesn't just put in words what the `InnerBuildArgs` does in code.

https://github.com/dotnet/runtime/pull/68424 may provide us with a more intuitive name which we can then use here.